### PR TITLE
Changes done for moving locators in common file for bnc scenarios

### PR DESCRIPTION
**Pull request by rkaur10**

### CODE REVIEW

File: common/browseFunctions.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    browser.moveToObject(browseLoc.searchBox);
+    browser.click(browseLoc.searchBox);
+    browser.setValue(browseLoc.searchBox, productID);
+    browser.moveToObject(browseLoc.goButton);
+    browser.click(browseLoc.goButton);
+}
+
+// No colour No Size Non FEAR ATB
+function pdpAtb(){
+browser.waitForVisible(browseLoc.pdpOpen);
```
Comment (SunilSyal) [inline common/browseFunctions.js]: fix alignment. provide some indent.

File: common/checkoutFunctions.js [hunk 2]
```diff
@@ -1,41 +1,95 @@
...
+}
+
+function payment() {
+  browser.click(checkoutLoc.continueToPayment);
+  browser.waitForVisible(checkoutLoc.paymentPageOpen);
+
+}
+function addCardForm() {
+  browser.selectByValue(checkoutLoc.cardType, '11101_M&S Credit Card');
+  browser.setValue(checkoutLoc.cardNumber, '5299301234567791');
```
Comment (SunilSyal) [inline common/checkoutFunctions.js]: indent is not consistent.

File: resources/checkoutLocator.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
+module.exports = {
+
+  //miniBag
+  header: '.header-link',
+  viewBag: '.secondary-btn',
+  checkout: '.btn-checkout',
+
+
+//checkoutLogin
+emailId: '#loginEmail',
```
Comment (SunilSyal) [inline resources/checkoutLocator.js]: line indent - global issue. Fix everywhere.

File: common/checkoutFunctions.js (hunk 2 above)
Comment (rkaur10) [reply common/checkoutFunctions.js]: done

File: resources/checkoutLocator.js (hunk 3 above)
Comment (rkaur10) [reply resources/checkoutLocator.js]: done

File: common/browseFunctions.js (hunk 1 above)
Comment (rkaur10) [reply common/browseFunctions.js]: done

**Pull request merged**